packages/gestalt-core/build.js

```diff
@@ -57,7 +57,14 @@ const cssModules = (options = {}) => {
         scopeNames[hash] = classnameBuilder.getMinifiedClassname(hash);
       }
 
-      return scopeNames[hash];
+      // if the DEVMODE environment variable is set, then don't minify the classnames
+      const isDevMode = process.env.DEVMODE === 'true';
```

Comment:
We already have `NODE_ENV`. Is there a reason to introduce another `env` field?

Reply:
Ah, re this: tweaking the NODE_ENV seems to cause a failure somewhere else. And it's not entirely obvious why

![image](https://github.com/pinterest/gestalt/assets/5509813/f70ddd40-8fb3-43c4-97ec-3eab246e025b)